crawl-ref/source/dat/des/branches/crypt.des

```diff
@@ -1463,6 +1486,884 @@ cG.MGM.GcG.1.GcG.1.GcG.1.Gc
 ccccccccccccccccccccccccccc
 ENDMAP
 
+NAME:    nicolae_crypt_booths_of_the_dead
+TAGS:    transparent
+DEPTH:   Crypt
+SHUFFLE: DE/FH, DE, FH
+SUBST:   D : x, E : ....x, F : xxxx., H : ., - = 0%...
+MAP
+xxxxxxxxxxxxxxxxxxxxxxxx
+x-...FF....HH....FF...-x
+x-...HH....FF....HH...-x
+xx..xxxx..xxxx..xxxx..xx
+x-..-xx-..-xx-..-xx-..-x
+x-..-xx-..-xx-..-xx-..-x
+xx..xxxx..xxxx..xxxx..xx
+x-...DD....EE....DD...-x
+x-...EE....DD....EE...-x
+xx..xxxx..xxxx..xxxx..xx
+@...-xx-..-xx-..-xx-...@
+@...-xx-..-xx-..-xx-...@
+xx..xxxx..xxxx..xxxx..xx
+x....EE....DD....EE...-x
+x....DD....EE....DD...-x
+xx..xxxx..xxxx..xxxx..xx
+x-..-xx-..-xx-..-xx-..-x
+x-..-xx-..-xx-..-xx-..-x
+xx..xxxx..xxxx..xxxx..xx
+x-...HH....FF....HH...-x
+x-...FF....HH....FF...-x
+xxxxxxxxxxxxxxxxxxxxxxxx
+ENDMAP
+
+# This one was inspired by scrolling past cheibrodos_crypt_draghalla and
+# misreading the name.
+NAME:    nicolae_crypt_church_of_dragula
+TAGS:    transparent
+DEPTH:   Crypt
+KITEM:   KD = |
```

Comment:
I was following gammafunk's house style: (K)ITEM -> (K)MONS -> (K)FEAT
y'all need to get together and work on a consistent one!
(Also, I hate using MONS directives: it's much easier for me to read a vault if the directive has the specified glyph right next to the monster definition, instead of having to count the commas.)